VisualPinball.Unity/VisualPinball.Unity/Game/Player.cs

```diff
@@ -51,6 +51,9 @@ public class Player : MonoBehaviour
 		[HideInInspector] [SerializeField] public string debugUiId;
 		[HideInInspector] [SerializeField] public string physicsEngineId;
 
+		[Tooltip("UpdateDuringGamplay")]

```

Comment:
Maybe something like "When enabled, update the switch, coil, lamp and wire manager windows in the editor (slower performance)"?

Reply:
Updated